setup.cfg

```diff
@@ -15,8 +15,26 @@ python = python
 universal = 0
 
 [flake8]
-builtins = _
+filename =
+    *.py
+    ./linkchecker
+builtins =
+    _
+    _n
```

Comment:
Oh, does this work?  flake8 documentation gave me the impression that these must be separated with commas, even if split on multiple lines:
- https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-filename
- https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-builtins

Reply:
Yes, before:
```
$ flake8 --select=F821
./linkcheck/director/console.py:38:15: F821 undefined name '_n'
./linkcheck/director/console.py:41:15: F821 undefined name '_n'
./linkcheck/director/console.py:43:15: F821 undefined name '_n'
./linkcheck/director/console.py:45:15: F821 undefined name '_n'
./linkcheck/logger/html.py:293:24: F821 undefined name '_n'
./linkcheck/logger/html.py:296:20: F821 undefined name '_n'
./linkcheck/logger/html.py:302:20: F821 undefined name '_n'
./linkcheck/logger/html.py:311:24: F821 undefined name '_n'
./linkcheck/logger/__init__.py:278:19: F821 undefined name '_'
./linkcheck/logger/__init__.py:340:16: F821 undefined name '_'
./linkcheck/logger/__init__.py:377:22: F821 undefined name '_'
./linkcheck/logger/__init__.py:380:22: F821 undefined name '_'
./linkcheck/logger/__init__.py:382:22: F821 undefined name '_'
./linkcheck/logger/__init__.py:390:22: F821 undefined name '_'
./linkcheck/logger/text.py:228:20: F821 undefined name '_n'
./linkcheck/logger/text.py:232:24: F821 undefined name '_n'
./linkcheck/logger/text.py:235:24: F821 undefined name '_n'
./linkcheck/logger/text.py:246:22: F821 undefined name '_n'
./linkcheck/logger/text.py:259:26: F821 undefined name '_n'
```

after:
```
$ flake8 --select=F821
./linkcheck/logger/__init__.py:278:19: F821 undefined name '_'
./linkcheck/logger/__init__.py:340:16: F821 undefined name '_'
./linkcheck/logger/__init__.py:377:22: F821 undefined name '_'
./linkcheck/logger/__init__.py:380:22: F821 undefined name '_'
./linkcheck/logger/__init__.py:382:22: F821 undefined name '_'
./linkcheck/logger/__init__.py:390:22: F821 undefined name '_'
```

Whether it is dangerous for the future I don't know. I guess flake8 has to do the same as distutils/setuptools.

Reply:
I was surprised to see these warnings remain, until I actually looked at the source and saw it do

```python
_ = lambda x: x
Fields = dict(
    realurl=_("Real URL"),
    cachekey=_("Cache key"),
...
    level=_("Level"),
    modified=_("Modified"),
)
del _
```

That'll be fun to untangle :/

Reply:
Actually, I think it's a flake8 (or, more precisely, pyflakes) bug.  If you have a global that shadows a builtin, and you delete the global, that makes the builtin available again.

Reply:
https://github.com/PyCQA/pyflakes/issues/548

Reply:
> [PyCQA/pyflakes#548](https://github.com/PyCQA/pyflakes/issues/548)

Subscribed!